ena-submission/src/ena_deposition/create_assembly.py

```diff
@@ -522,11 +536,14 @@ def assembly_table_create(
                 tries += 1
             if number_rows_updated == 1:
                 logger.info(
-                    f"Assembly submission for accession {row['accession']} succeeded! - waiting for ENA accession"
+                    f"Assembly submission for accession {row['accession']} succeeded with: {assembly_creation_results.result}"
                 )
         else:
             update_assembly_error(
-                db_config, assembly_creation_results.errors, seq_key=row, update_type="creation"
+                db_config,
+                str(assembly_creation_results.errors),
```

Comment:
ah this is actually a valid type issue (sorry my bad) - `assembly_creation_results.errors` is a list of errors so converting it so a str might be a bit problematic - I will try to add a commit to clean this up a bit - thanks for flagging this!

Reply:
You sometimes pass a list[str] sometimes a [str] - in the end you just dump it to string so there's nothing problematic.